data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/DataFileLoader.java

```diff
@@ -128,7 +150,9 @@ public void run() {
         int lineCount = 0;
         int lastLineProcessed = 0;
 
-        try (GZIPInputStream gzipInputStream = new GZIPInputStream(s3ObjectReader.readFile(bucketName, key))) {
+        try {
+            InputStream inputStream = objectReader.readFile(bucketName, key);
```

Comment:
```
nit:
try (
  InputStream inputStream = objectReader.readFile(bucketName, key);
  GZIPInputStream gzipInputStream = new GZIPInputStream(inputStream);
  BufferedReader reader = new BufferedReader(new InputStreamReader(gzipInputStream));
) {
..
}
```

Reply:
I didn't know we could do that. I thought it can only be one.